<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupOutput</class>
 <widget class="QWidget" name="setupOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Output Options:</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select at least one output for the simulation to produce. All generated files will appear in the selected working directory.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>560</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Simulation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>560</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>751</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>711</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>711</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>711</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>691</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,12" rowminimumheight="90,90,90,90">
      <item row="0" column="0">
       <widget class="QCheckBox" name="heatmapCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">left-margin:50%;
right-margin:50%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="csvCheck">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="configCheck">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="popCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="3,1">
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <widget class="QRadioButton" name="heatAll">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>310</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>For each year</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="heatFinal">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>310</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Final year only</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_3" native="true">
          <widget class="QLabel" name="heatDescription">
           <property name="geometry">
            <rect>
             <x>2</x>
             <y>0</y>
             <width>296</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Generate population depletion heatmap(s)</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="heatBackrounds">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>307</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Use map background if avaliable (WIP)</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="heatScale">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>221</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Scale output Resolution (integer):</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="heatScaleValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>30</y>
             <width>48</width>
             <height>24</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
          <widget class="QFrame" name="frame_2">
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>10</y>
             <width>141</width>
             <height>71</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="sizeLabel">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>121</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Output Size (px):</string>
            </property>
           </widget>
           <widget class="Line" name="line_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>111</width>
              <height>21</height>
             </rect>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QLabel" name="widthDisp">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>131</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Width:</string>
            </property>
           </widget>
           <widget class="QLabel" name="heightDisp">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>131</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Height:</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="csvDescription">
          <property name="text">
           <string>Generate CSV text file(s) for the population depletion grid</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <widget class="QRadioButton" name="csvAll">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>275</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>For each year</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QRadioButton" name="csvFinal">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>275</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Final year only</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="popDescription">
          <property name="text">
           <string>Output a configuration file that saves the parameters of this simulation. (WIP)</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="configPopCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Include population data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="popOnlyDescription">
          <property name="text">
           <string>Output a seperate population only data file. (WIP)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>line</zorder>
    <zorder>line_3</zorder>
    <zorder>line_2</zorder>
    <zorder>gridLayoutWidget</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
